### DISCUSSION TITLE
Password to run local?

### DISCUSSION THREAD

**Discussion by joesalvati68** [llmware-ai/llmware#231] [Q&A]
I am trying to run this local and when I go to install it asks for a password but I don't see mention of a password.  What am I missing?

**Comment by doberst** [comment]
@joesalvati68 - sorry for the slow response - what specifically are you running?   If it is a local model, then there should not be any passwords required (including huggingface) - if you share more details, then we can take a look and help to debug.....

**Reply by joesalvati68** [reply]
That's the thing.  I'm not using anything.  I downloaded it right from Github

**Comment by joesalvati68** [comment]
Can I take a screenshot and share?

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by doberst** [comment]
Yes, please go ahead!

(Btw, happy new year!)   :)

**Comment by joesalvati68** [comment]
Same to you.  Will do

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
